test/testdata/lsp/code_actions/move_method/instance_method.rb

```diff
@@ -0,0 +1,18 @@
+# typed: strict
+# selective-apply-code-action: refactor.extract
+#
+# No code actions should be available for instance methods
```

Comment:
I'm not super familiar with the test runner for code actions. Would this test start to fail if some sort of support for instance methods were added eventually?